project/Release.scala

```diff
@@ -1,62 +1,20 @@
 import sbt._
 import Keys._
-import StatusPlugin.autoImport._
-import org.apache.ivy.util.url.CredentialsStore
 import com.typesafe.sbt.JavaVersionCheckPlugin.autoImport._
+import _root_.bintray.BintrayPlugin.autoImport._
+import _root_.bintray.InternalBintrayKeys._
 
-object Release extends Build {
-  lazy val remoteBase = SettingKey[String]("remote-base")
-  lazy val remoteID = SettingKey[String]("remote-id")
+object Release {
   lazy val launcherRemotePath = SettingKey[String]("launcher-remote-path")
   lazy val deployLauncher = TaskKey[Unit]("deploy-launcher", "Upload the launcher to its traditional location for compatibility with existing scripts.")
-  lazy val checkCredentials = TaskKey[Unit]("checkCredentials", "Checks to ensure credentials for this user exists.")
```

Comment:
bintray-sbt already has checkCredentials right?  Would be sad to loose this kind of check before releasing.